### PR TITLE
full form example

### PR DESCRIPTION
**Pull request by lemingos**
Full form example and simple form exmple.

### CODE REVIEW

**Comment by pavelloz** [comment]
Can you update code to correspond with code snippets @ docs? (quotes, "hello" messages)

File: marketplace_builder/custom_model_types/car.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Car
+custom_attributes:
```
Comment (Slashek) [inline marketplace_builder/custom_model_types/car.yml]: you can use `properties:`

File: marketplace_builder/custom_model_types/example_model.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: example_model
+properties:
```
Comment (Slashek) [inline marketplace_builder/custom_model_types/example_model.yml]: like here;)

File: marketplace_builder/form_configurations/car_form.liquid [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: "car_form"
```
Comment (Slashek) [inline marketplace_builder/form_configurations/car_form.liquid]: no need `""`

File: marketplace_builder/form_configurations/full_form_example.liquid [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+name: full_form_example
+resource: example_model
+resource_owner: anyone
+return_to: '/full-form-example'
+flash_alert: "Hello from flash alert"
+flash_notice: "Hello from flash notice"
+spam_protection: "" # e.g. recaptcha - needs to be fonfigured
```
Comment (Slashek) [inline marketplace_builder/form_configurations/full_form_example.liquid]: fonfigured -> configured

File: marketplace_builder/views/pages/full-form-example.liquid [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+slug: full-form-example
+---
+
```
Comment (Slashek) [inline marketplace_builder/views/pages/full-form-example.liquid]: ✂️ blank line

File: marketplace_builder/custom_model_types/car.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+name: Car
```
Comment (Slashek) [inline marketplace_builder/custom_model_types/car.yml]: probably best to use snake case here

File: marketplace_builder/custom_model_types/car.yml (hunk 1 above)
Comment (pavelloz) [reply marketplace_builder/custom_model_types/car.yml]: you can use suggest mode to make CR process faster and easier @Slashek  :)

```suggestion
properties:
```

**Comment by pavelloz** [comment]
Anyway, some comments from https://github.com/mdyd-dev/nearme-documentation/pull/304 apply here, since they should be in sync

**Pull request merged**